sdk/identity/identity/src/tokenCache/TokenCachePersistence.ts

```diff
@@ -31,11 +32,20 @@ export class TokenCachePersistence {
       retryDelay: 50
     };
 
+    let extensions: any;
+    try {
+      /* eslint-disable-next-line @typescript-eslint/no-require-imports */
+      extensions = require("@azure/msal-node-extensions");
+    } catch (e) {
+      throw new CredentialUnavailable(
+        "Token cache persistence requires the optional dependency '@azure/msal-node-extensions' to work correctly"
```

Comment:
It is not clear to the user what the next step is...

```suggestion
        "To use the token cache persistence feature, please install the package '@azure/msal-node-extensions@1.0.0-alpha.6'."
```

Reply:
I like this! Thank you!